File: src/main/java/com/github/thaynarasilvapinto/SimuladorBanco/controller/request/ClienteCriarRequest.java
package com.github.thaynarasilvapinto.SimuladorBanco.controller.request;

import javax.validation.constraints.NotNull;

public class ClienteCriarRequest {
    @NotNull
    private String nome;
    @NotNull
    private String cpf;

    public ClienteCriarRequest() {
    }

    public ClienteCriarRequest(String nome, String cpf) {
        this.nome = nome;
        this.cpf = cpf;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getCpf() {
        return cpf;
    }

    public void setCpf(String cpf) {
        this.cpf = cpf;
    }
}

File: src/main/kotlin/com/github/thaynarasilvapinto/simuladorbanco/controller/request/ClienteCriarRequest.kt
package com.github.thaynarasilvapinto.simuladorbanco.controller.request

import org.hibernate.validator.constraints.br.CPF
import javax.validation.constraints.NotNull

data class ClienteCriarRequest(
        @get: NotNull
        val nome: String?,
        @get: NotNull
        @get: CPF
        val cpf: String?
)